rust_snuba/src/lib.rs

```diff
@@ -7,6 +7,20 @@ use pyo3::prelude::*;
 
 #[pymodule]
 fn rust_snuba(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
+    // let DSN
+    let sentry_dsn = Python::with_gil(|py| {
+        let snuba_settings = PyModule::import(py, "snuba.settings")?;
+        let dsn: Result<String, PyErr> = snuba_settings
+            .getattr("SENTRY_DSN")?
+            .extract();
+        dsn
+    }).unwrap();
```

Comment:
SENTRY_DSN is optional since not all environments send errors to Sentry. We should probably skip the Sentry initialization if it's None rather than crash.

Reply:
Updated the func. It should be safe now, but maybe using py import is overkill and I should just export the DSN to an environment variable and fetch it from Rust?